examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -79,10 +79,11 @@ echo "    Slurm output dir: $ROOT_DIR"
 cp buildenv/submit.daint.slurm .
 sed s/\<NAME\>/standalone/g submit.daint.slurm -i
 sed s/\<NTASKS\>/$ranks/g submit.daint.slurm -i
-sed s/\<NTASKSPERNODE\>/$ranks/g submit.daint.slurm -i
-sed s/\<CPUSPERTASK\>/1/g submit.daint.slurm -i
-sed s/#SBATCH\ --output=\<OUTFILE\>//g submit.daint.slurm -i
+sed s/\<NTASKSPERNODE\>/1/g submit.daint.slurm -i
+sed s/\<CPUSPERTASK\>/12/g submit.daint.slurm -i
```

Comment:
You can get the 12 out of the `machineEnvironment.sh` afaik.

Reply:
Should you not also set `#SBATCH --cpus-per-task=12` and add `export OMP_NUM_THREADS=$SLURM_CPUS_PER_TASK` to the CMD to enable OpenMP thread placement?

Reply:
The template has the line `export OMP_NUM_THREADS=<CPUSPERTASK>` already in there, we sed both at the same time

Reply:
the buildenv sets it to 8. Do we know why?

Reply:
Oh gosh. Not a clue. Possibly abused for `make -j ${nthreads}` and thus limited to a reasonable number? Conclusion from my side: don't use `machineEnvironment.sh`.

Reply:
ok